openfecwebapp/api_caller.py

```diff
@@ -86,14 +86,19 @@ def load_legal_advisory_opinion(ao_no):
     if not (documents and len(documents)):
         return None
 
-    canonical_document = documents[0]
+    for document in documents:
+        canonical_document = document
+        if (document['category'] == 'Final Opinion'):
```

Comment:
Added test.  It should never execute as there's a test for the `documents` array already;  but better safe than sorry.